cube_time: 0.8
cone_time: 1.6
server_timeout: 20.0
path_zero_timeout: 3.0
current_threshold: 150.0 # some arbitrary unit
joint: intake_leader # for current sensing
time_before_reverse: 0.0

intake_speed: 0.5
outtake_speed: 0.8
intake_small_speed: 0.05